test/create_env_cluster_topics_test.go

```diff
@@ -101,5 +101,8 @@ func TestEnvClusterTopic(t *testing.T) {
 
 		output = terraform.Output(t, runOptions, "connector_gcs_sink_connector_id")
 		a.NotEmpty(output)
+
+		output = terraform.Output(t, runOptions, "kafka_acl_consumer_group_id")
```

Comment:
I don't understand this - we removed the ACL, so we should be testing it for it being missing, not present?